concepticondata/concepticon.tsv

```diff
@@ -3000,3 +3000,25 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3004	FORMER	Time	Coming before in time.	Property	
 3005	FAMILIAR	Emotions and values	Knowing somebody or something well.	Property	
 3006	SPACE (AVAILABLE)	Spatial relations	Available space to place things.	Person/Thing	
+3007	EUCALYPT	Agriculture and vegetation	Tree or wood for plants with capsule fruiting bodies belonging to closely related genera found across Australia.	
```

Comment:
Please provide the basic word class information for all added concepts, that is person/thing, etc. The ontology is simple: person/thing is just  = noun, property is adjective, use other for things where you are not sure.

Reply:
Sorry, had planned to include it and forgot.